src/prefect/utilities/executors.py

```diff
@@ -71,68 +73,22 @@ def multiprocessing_timeout(
         - the result of `f(*args, **kwargs)`
 
     Raises:
-        - AssertionError: if run from a daemonic process
         - TimeoutError: if function execution exceeds the allowed timeout
     """
-
     if timeout is None:
         return fn(*args, **kwargs)
 
-    def retrieve_value(
-        *args: Any, _container: multiprocessing.Queue, **kwargs: Any
-    ) -> None:
-        """Puts the return value in a multiprocessing-safe container"""
-        try:
-            _container.put(fn(*args, **kwargs))
-        except Exception as exc:
-            _container.put(exc)
-
-    q = multiprocessing.Queue()  # type: multiprocessing.Queue
-    kwargs["_container"] = q
-    p = multiprocessing.Process(target=retrieve_value, args=args, kwargs=kwargs)
-    p.start()
-    p.join(timeout)
-    p.terminate()
-    if not q.empty():
-        res = q.get()
-        if isinstance(res, Exception):
-            raise res
-        return res
-    else:
-        raise TimeoutError("Execution timed out.")
+    executor = ThreadPoolExecutor(max_workers=2)
```

Comment:
Just curious (and for my clarification), why `2`?

Reply:
Good question; I don't know why I did that.  I probably should just let it use the default - good catch.